lib/client/identityfile/identity_test.go

```diff
@@ -127,7 +128,7 @@ func TestWrite(t *testing.T) {
 	// test OpenSSH-compatible identity file creation:
 	cfg.OutputPath = filepath.Join(outputDir, "openssh")
 	cfg.Format = FormatOpenSSH
-	_, err := Write(cfg)
+	_, err := Write(context.TODO(), cfg)
```

Comment:
Nit: Can we use `context.Background()` in these tests instead? While it's the same under the hood, `context.TODO()` is meant to be used in places where we want to pass a proper context in the future. Tests will always use `context.Background` though.